examples/cluster/cluster.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
 spec:
   clusterNetwork:
     pods:
-      cidrBlocks: ["192.168.0.0/16"]
+      cidrBlocks: ["${CLUSTER_CIDR}"]
```

Comment:
Should this potentially support setting both ipv4 and ipv6 CIDRs?

Reply:
yeah, but that's dual stack and is pretty alpha status, I'd prefer to take one step at a time.
Have an IPv6 cluster (the hosts will be dual stack but the cluster single stack) and once it works we can move forward

Reply:
sgtm, thanks for clarifying